### PR TITLE
Update cert-generator and certs

### PR DESCRIPTION
**Pull request by Manishearth**
r? @avadacatavra

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15106)

### CODE REVIEW

**Comment by Manishearth** [comment]
JSON of the certificate list: https://gist.github.com/Manishearth/125b162c40f10cdb2800acb292eb9119

I wonder why LetsEncrypt isn't in there.

**Comment by avadacatavra** [comment]
@Manishearth i'll review once we figure out why lets encrypt isn't there :)

**Comment by nox** [comment]
What's the status on this?

**Comment by Manishearth** [comment]
We don't know why LE isn't there.

**Comment by Manishearth** [comment]
I guess landing this is strictly better? The old code doesn't work at all.

**Comment by jdm** [comment]
Have we talked to anybody from the Gecko team who would know?

**Comment by Manishearth** [comment]
I think we did, but not much, and there wasn't a conclusion. We should look at this again.

**Comment by nox** [comment]
Did we look at this again?

**Comment by Manishearth** [comment]
No

**Comment by nox** [comment]
@Manishearth Could you?

**Comment by Manishearth** [comment]
I don't care enough at this point. This can be merged as-is with a disclaimer that it doesn't get all certs.

**Comment by ferjm** [comment]
> I wonder why LetsEncrypt isn't in there.
AFAIK LetsEncrypt certificates are cross-signed by IdenTrust ("DST Root CA X3" Root CA), which is listed in the json

**Comment by Manishearth** [comment]
We ran it on a Firefox with LE in the trust store and it didn't work.

**Comment by Manishearth** [comment]
Let me try again.

**Comment by Manishearth** [comment]
I had a second look, looks like LE isn't in the trust store. Could swear it was.

Updated with nightly cert store. See https://gist.github.com/Manishearth/84049982d60d6579bfc0eaf5961300d4 

Anyway, r? @avadacatavra

**Comment by jdm** [comment]
@bors-servo: retry
* #16249
